apk/APKBUILD

```diff
@@ -25,6 +25,8 @@ build() {
     cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo \
           -DENABLE_BACKTRACE:BOOL=ON \
           -DENABLE_DIST:BOOL=ON \
+          -DLUAJIT_DISABLE_SYSPROF:BOOL=ON \
+          -DLUAJIT_ENABLE_GC64=${GC64:=OFF} \
```

Comment:
Should we add `:BOOL` here?

I already wrote about it. I don't know why you ignore my comments.

Reply:
OK